.github/workflows/ci.yml

```diff
@@ -0,0 +1,94 @@
+name: CI
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  push:
+    branches:
+      - master
+env:
+  IMAGE_TAG: ${{ github.sha }}
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/cache@v1
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-yarn-client-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-client-
+      - name: yarn
+        run: yarn
+      - name: Lint
+        run: yarn lint
+      - name: Unit Test
+        run: yarn test
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/cache@v1
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-yarn-client-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-client-
+      - name: yarn
+        run: yarn
+      - name: Build
+        run: make build
+      - name: Save Docker images
+        run: |
+          mkdir -p docker-image
+          docker save -o docker-image/reviewer-client.tar libero/reviewer-client:${IMAGE_TAG}
+          gzip docker-image/reviewer-client.tar
```

Comment:
this seems to cut the image size to upload to 1/3 of the original size. Artifacts are available to download in a [`.zip` file](https://github.com/libero/reviewer-client/suites/353513219/artifacts/652471) but it doesn't seem compression would be going on between the actions if we don't make this explicit. Do you confirm?

Reply:
This was a copy-paste job of https://github.com/libero/article-store/blob/0890b80f030b60a29bd60cf383718455c4f65014/.github/workflows/ci.yml#L33 can remove if it's not appropriate in this case. Would be good to get the article stores use of compression explained?

Reply:
I think this is the correct thing to do regardless of what Github Actions is doing - as zipping here reduces network traffic

Reply:
If you have not discovered anything new, continue gzipping as we did elsewhere.
(`bzip2` was not worth the CPU time)

Reply:
> regardless of what Github Actions is doing

(if they zipped it already, it would be a waste of CPU; but they don't as far as we know)